usecases/MinimumWorkingExample/dodo.py

```diff
@@ -87,6 +99,31 @@ def task_extract_processed_data_emodul():
             }
 
 
+#extract standardized metadata for the mixture
+def task_extract_metadata_mixture():
+    # create folder, if it is not there
+    Path(metadata_mixture_directory).mkdir(parents=True, exist_ok=True)
+
+    # setting for fast test, defining the list
+    if config['mode'] == 'cheap':
```

Comment:
see above, IMO no need to have a cheap example here (for the E this is primarily for the parameter estimation, the extraction is also there quite straightforward)